operator/zitadel/kinds/iam/zitadel/migration/done.go

```diff
@@ -1,15 +1,16 @@
 package migration
 
 import (
+	"time"
+
 	"github.com/caos/orbos/mntr"
 	"github.com/caos/orbos/pkg/kubernetes"
 	"github.com/caos/zitadel/operator"
 	"github.com/pkg/errors"
-	"time"
 )
 
 const (
-	timeout time.Duration = 300
+	timeout = 20 * time.Minute
```

Comment:
I think this my own change from the cr12-02 branch? Does this still make sense?

Reply:
yeah unfortunately the migration job still needs 5-10 min to complete